### PR TITLE
added wind direction icon

### PR DESCRIPTION
**Pull request by Cutler7**

### CODE REVIEW

File: src/assets/icons/fa-wind-direction.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (Duelsik) [inline src/assets/icons/fa-wind-direction.svg:3]: Remove id

File: src/components/DailyWeather/DailyWeatherDetails.tsx [hunk 2]
```diff
@@ -58,11 +59,22 @@ const ArrowRightIconWrapperRestyled = styled(ArrowRightIconWrapper)`
...
+const WindDirectionWrapper = styled.span`
+  position: relative;
+`;
+
+const WindDirectionIconStyled = styled(WindDirectionIcon)`
+  position: absolute;
+  transform: ${({ deg }) => `rotate(${deg}deg)`};
+  height: 1.5em;
+  right: 0.2em;
+`;
```
Comment (Duelsik) [inline src/components/DailyWeather/DailyWeatherDetails.tsx:71]: Kinda odd way of placing inline icon, but let's do that in future.

**Review by Duelsik — CHANGES_REQUESTED** [review]
Small detail.

File: src/components/DailyWeather/DailyWeatherDetails.tsx (hunk 2 above)
Comment (Cutler7) [reply src/components/DailyWeather/DailyWeatherDetails.tsx:71]: it was hard to place this icon in the right place

File: src/assets/icons/fa-wind-direction.svg (hunk 1 above)
Comment (Cutler7) [reply src/assets/icons/fa-wind-direction.svg:3]: done

**Review by Duelsik — APPROVED** [review]

**Pull request merged**